opentelemetry-appender-log/Cargo.toml

```diff
@@ -17,3 +17,7 @@ log = {version = "0.4.17", features = ["kv_unstable", "std"]}
 [features]
 logs_level_enabled = ["opentelemetry/logs_level_enabled"]
 default = ["logs_level_enabled"]
+
+[dev-dependencies]
+opentelemetry_sdk = { path = "../opentelemetry-sdk", features = [ "testing", "logs"] }
```

Comment:
Can you once validate building the in-memory example. I suspect `logs_level_enabled` feature needs to be enabled for `opentelemetry-sdk` as it is enabled by-default for `opentelemetry-appender-log`.

Reply:
Corrected, the features, thanks for pointing out.